github/github.go

```diff
@@ -478,12 +478,6 @@ func (c *Client) Do(ctx context.Context, req *http.Request, v interface{}) (*Res
 		return nil, err
 	}
 
-	defer func() {
-		// Drain up to 512 bytes and close the body to let the Transport reuse the connection
-		io.CopyN(ioutil.Discard, resp.Body, 512)
-		resp.Body.Close()
```

Comment:
I believe that we should go back to how this looked prior to the addition of the drain:
https://github.com/google/go-github/commit/e0ff7111b024fda991e457d975e0b73e1380f05f

Specifically, this should now be:

```go
defer resp.Body.Close()
```

Note that this is an HTTP client, and [its documentation](https://godoc.corp.google.com/pkg/net/http) says:

```
The client must close the response body when finished with it:
...<example elided>...
```